lib/node_modules/@stdlib/math/base/special/fast/alpha-max-plus-beta-min/test/test.native.js

```diff
@@ -0,0 +1,68 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var randu = require( '@stdlib/random/base/randu' ).factory;
+var abs = require( '@stdlib/math/base/special/abs' );
+var hypot = require( '@stdlib/math/base/special/hypot' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+var approx = tryRequire( resolve( __dirname, './../lib/native.js' ) );
```

Comment:
```suggestion

// VARIABLES //

var hypot = tryRequire( resolve( __dirname, './../lib/native.js' ) );
var opts = {
	'skip': ( hypot instanceof Error )
};


```

Reply:
You may refer https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/fast/abs/test/test.native.js